include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -372,6 +392,15 @@ struct Env : detail::Base<OrtEnv> {
   Env& CreateAndRegisterAllocator(const OrtMemoryInfo* mem_info, const OrtArenaCfg* arena_cfg);  ///< Wraps OrtApi::CreateAndRegisterAllocator
 };
 
+
+
+namespace detail {
+
+// Logs a message at the given severity level. Typically called via the ORT_CXX_LOG macro.
+void LogImpl(OrtLoggingLevel log_severity_level, const char* message, const char* file_path, int line_number,
```

Comment:
> LogImpl(OrtLoggingLe [](http://example.com/codeflow?start=4&length=21)

Should the macro be defined some place very near this?

Reply:
Moved macro next to implementation function.